djangocms_version_locking/cms_config.py

```diff
@@ -9,7 +9,7 @@
 class VersionLockingCMSExtension(CMSAppExtension):
 
     def __init__(self):
-        # The monkey patch is here to be sure thta at module load time the Version class
+        # The monkey patch is here to be sure that at module load time the Version class
         # is registered and can be overriden without requiring a strict load order
         # in the INSTALLED_APPS setting in a projects settings.py
         from .monkeypatch import admin as monkeypatch_admin
```

Comment:
is there a specific reason this needs to be called here rather than from VersionLockingConfig.ready? just trying to see one single place to call all monkey patches.

Reply:
the comment is fine. it was more for my understanding. 👍